.github/workflows/tox-pytest.yml

```diff
@@ -0,0 +1,73 @@
+name: tox-pytest
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Compile Zenodo Datastore DOIs
+      run: grep -e '.*10\.\(5281\|5072\)/zenodo\..*' src/pudl/workspace/datastore.py | sed -e 's/",*$//g' | sed -e 's/^.*"//g' | sort > datastore-dois.txt
+
+    - uses: actions/cache@v2
+      id: cache-zenodo-datastore
+      with:
+        path: ~/pudl-work/data/
+        key: zenodo-datastore-${{ hashFiles('datastore-dois.txt') }}
+        restore-keys: |
+          zenodo-datastore-${{ hashFiles('datastore-dois.txt') }}
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get install libsnappy-dev
+        python -m pip install --upgrade pip
+        pip install tox
+
+    - name: Check for cached data
+      run: |
+        mkdir -p ~/pudl-work/data/
+        find ~/pudl-work/data/
```

Comment:
are you making and then finding after that just to be sure it was made?

Reply:
`mkdir -p` makes a directory if it doesn't exist, but doesn't complain if it does exist already. So I'm maybe making the directory, and then definitely asking it to list all of the files that exist in there. If the data was cached, it'll show up, if it wasn't cached, it won't.